docs/features/Mappings.md

```diff
@@ -336,6 +337,7 @@ The same is achieved with [string interpolation](https://twig.symfony.com/doc/1.
   "unset": ["color"]
 }
 ```
+{% endraw %}
```

Comment:
{% raw %} {% endraw %} can be removed

Reply:
The google doc isn't up to date, but i will remove the raw & endraw from there as well 👍

Reply:
I updated the doc and removed that already, maybe someone did paste the coreBundle mapping.md in the doc again